src/javascripts/components/search.tracking.mjs

```diff
@@ -25,16 +25,18 @@ export function trackConfirm(searchQuery, searchResults, result) {
 
   addToDataLayer({
     event: 'site_search',
-    eventDetails: {
-      category: 'site search',
+    eventData: {
       action: 'click',
-      label: `${searchTerm} | ${result.title}`
-    },
+      text: searchTerm,
+      section: result.title
+    }
+  })
+
+  addToDataLayer({ ecommerce: null })
```

Comment:
Non blocking question: is the idea that this is resetting the `ecommerce` value for the `select_item` event below?

Regardless of if I've got that right I think it'd be good to add a comment explaining what this line is doing. Same story for line 74.

Reply:
Yes, I'll add a comment about this.